iree/turbine/ops/iree.py

```diff
@@ -151,6 +167,53 @@ def generate(self, ksel: KernelSelection, kb: KernelBuilder):
         kb.yield_results(result)
 
 
+################################################################################
+# IREE device affinity to torch device map
+################################################################################
```

Comment:
I am not sure if I should move this section somewhere else. For example in [iree/turbine/runtime/device.py](https://github.com/iree-org/iree-turbine/blob/66830917b9be8253d83f80abe4dcccfbe8cc6352/iree/turbine/runtime/device.py).